x-pack/plugins/endpoint/scripts/resolver_generator.ts

```diff
@@ -91,6 +91,13 @@ async function main() {
       type: 'number',
       default: 30,
     },
+    maxChildrenPerNode: {
```

Comment:
This makes the children of a tree more deterministic. This is helpful for tests that want to know exactly how many children each node will have.